### ISSUE TITLE
Cant install: error notarget In most cases you or one of your dependencies are requesting 20 error notarget a package version that doesn't exist.

### ISSUE THREAD

**Issue by GAZ082** [Hypnos3/node-red-contrib-cast#33]
Hey there. Trying to install from ND and I get this:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node',
1 verbose cli   '/usr/bin/npm',
1 verbose cli   'install',
1 verbose cli   '--save',
1 verbose cli   '--save-prefix=~',
1 verbose cli   '--production',
1 verbose cli   'node-red-contrib-cast@0.2.3' ]
2 info using npm@5.6.0
3 info using node@v8.11.4
4 verbose npm-session 2b46e81277b82ec2
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 200 https://registry.npmjs.org/node-red-contrib-cast 2279ms
8 silly registry:manifest no matching version for node-red-contrib-cast@0.2.3 in the cache. Forcing revalidation
9 http fetch GET 304 https://registry.npmjs.org/node-red-contrib-cast 386ms (from cache)
10 silly fetchPackageMetaData error for node-red-contrib-cast@0.2.3 No matching version found for node-red-contrib-cast@0.2.3
11 verbose type version
12 verbose stack node-red-contrib-cast: No matching version found for node-red-contrib-cast@0.2.3
12 verbose stack     at pickManifest (/usr/lib/node_modules/npm/node_modules/pacote/node_modules/npm-pick-manifest/index.js:65:11)
12 verbose stack     at fetchPackument.then.packument (/usr/lib/node_modules/npm/node_modules/pacote/lib/fetchers/registry/manifest.js:52:18)
12 verbose stack     at tryCatcher (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
12 verbose stack     at Promise._settlePromiseFromHandler (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:512:31)
12 verbose stack     at Promise._settlePromise (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:569:18)
12 verbose stack     at Promise._settlePromise0 (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
12 verbose stack     at Promise._settlePromises (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:693:18)
12 verbose stack     at Async._drainQueue (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:133:16)
12 verbose stack     at Async._drainQueues (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:143:10)
12 verbose stack     at Immediate.Async.drainQueues (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:17:14)
12 verbose stack     at runCallback (timers.js:810:20)
12 verbose stack     at tryOnImmediate (timers.js:768:5)
12 verbose stack     at processImmediate [as _immediateCallback] (timers.js:745:5)
13 verbose cwd /home/pi/.node-red
14 verbose Linux 4.14.79+
15 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "--save" "--save-prefix=~" "--production" "node-red-contrib-cast@0.2.3"
16 verbose node v8.11.4
17 verbose npm  v5.6.0
18 error code ETARGET
19 error notarget No matching version found for node-red-contrib-cast@0.2.3
20 error notarget In most cases you or one of your dependencies are requesting
20 error notarget a package version that doesn't exist.
21 verbose exit [ 1, true ]

```

Any help is appreciated!

**Comment by Failure404** [comment]
I am experiencing the same problem when trying to update from 0.2.2 to 0.2.3.
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for node-red-contrib-cast@0.2.3
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

**Comment by sismodico** [comment]
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'audit', 'fix' ]
2 info using npm@6.5.0
3 info using node@v10.15.0
4 verbose config Skipping project config: /home/pi/.npmrc. (matches userconfig)
5 verbose npm-session b0248143659a7e29
6 verbose stack Error: No package.json found: Cannot audit a project without a package.json
6 verbose stack     at Bluebird.all.spread (/usr/lib/node_modules/npm/lib/audit.js:143:19)
6 verbose stack     at tryCatcher (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
6 verbose stack     at Promise._settlePromiseFromHandler (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:509:35)
6 verbose stack     at Promise._settlePromise (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:569:18)
6 verbose stack     at Promise._settlePromise0 (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
6 verbose stack     at Promise._settlePromises (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:694:18)
6 verbose stack     at Promise._fulfill (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:638:18)
6 verbose stack     at PromiseArray._resolve (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise_array.js:126:19)
6 verbose stack     at PromiseArray._promiseFulfilled (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise_array.js:144:14)
6 verbose stack     at PromiseArray._iterate (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise_array.js:114:31)
6 verbose stack     at PromiseArray.init [as _init] (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise_array.js:78:10)
6 verbose stack     at Promise._settlePromise (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:566:21)
6 verbose stack     at Promise._settlePromise0 (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
6 verbose stack     at Promise._settlePromises (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:694:18)
6 verbose stack     at Promise._fulfill (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:638:18)
6 verbose stack     at PromiseArray._resolve (/usr/lib/node_modules/npm/node_modules/bluebird/js/release/promise_array.js:126:19)
7 verbose cwd /home/pi
8 verbose Linux 4.14.79+
9 verbose argv "/usr/bin/node" "/usr/bin/npm" "audit" "fix"
10 verbose node v10.15.0
11 verbose npm  v6.5.0
12 error code EAUDITNOPJSON
13 error audit No package.json found: Cannot audit a project without a package.json
14 verbose exit [ 1, true ]

**Comment by sismodico** [comment]
anche io  riscontro un problm nell'installazione di questo nodo sia da riga dicomando che dall   gestione dellle pallette  da node-red

**Comment by Hypnos3** [comment]
Can you check Version 0.2.4? I have removed lot of dependencies there.

**Comment by GAZ082** [comment]
It works now. You can close the issue. Thanks!

**Comment by Hypnos3** [comment]
fixed with 0.2.4
